tests/integration/docusaurus/connecting_to_your_data/how_to_connect_to_sqlite_data.py

```diff
@@ -0,0 +1,80 @@
+import pathlib
+import sqlite3
+
+
+def get_sqlite_database_path() -> pathlib.Path:
+    relative_path = pathlib.Path(
+        "..",
+        "..",
+        "..",
+        "test_sets",
+        "taxi_yellow_tripdata_samples",
+        "sqlite",
+        "yellow_tripdata.db",
+    )
+    return pathlib.Path(__file__).parent.joinpath(relative_path).resolve(strict=True)
+
+
+sqlite_database_path = get_sqlite_database_path()
+assert sqlite_database_path.is_file()
+
+
+def get_table_name(sqlite_database_path: pathlib.Path) -> list:
```

Comment:
@TrangPham I am not sure if this way of getting a table name is from another example; in my opinion, it is better to just open this file with one of the DB Workstation tools (e.g., I use "DBeaver") and look up the table name.  What do @anthonyburdi and @Shinnnyshinshin think?  Thanks.